gait:
    pantograph_leg: false
    max_linear_velocity_x: 0.5
    max_linear_velocity_y: 0.25
    max_angular_velocity_z: 1.0
    stance_duration: 0.25
    swing_height: 0.04
    stance_depth: 0.0
    nominal_height: 0.20

stance_phase_signal:
    - [1, 1, 1, 0]     #0
    - [1, 1, 0.5, 0.5] #0.25
    - [1, 1, 0, 1]     #0.5
    - [0.5, 0.5, 0, 1] #0.75
    - [0, 0, 0, 1]     #1
    - [0, 0, 0.5, 0.5] #0.25
    - [0, 0, 1, 0]     #0.5
    - [0.5, 0.5, 1, 0] #0.75

swing_phase_signal:
    - [0, 0, 0, 1]     #0
    - [0, 0, 0.5, 0.5] #0.25
    - [0, 0, 1, 0]     #0.5
    - [0.5, 0.5, 1, 0] #0.75
    - [1, 1, 1, 0]     #1
    - [1, 1, 0.5, 0.5] #0.25
    - [1, 1, 0, 1]     #0.5
    - [0.5, 0.5, 0, 1] #0.75 